### PR TITLE
update installation page

### PR DESCRIPTION
**Pull request by johnnychen94**
preview: https://juliaimages.org/previews/PR115/install

### CODE REVIEW

File: docs/src/install.md [hunk 1]
```diff
@@ -1,57 +1,72 @@
...
     People in some regions such as China might fail to install/precompile `Images` due to poor network
     status. Using proxy/VPN that has stable connection to Amazon S3 and Github can solve this issue.
 
 ## Loading your first image
 
-When testing ideas or just following along with the documentation, it can be
-useful to have some images to work with.
-The [TestImages](https://github.com/JuliaImages/TestImages.jl) package bundles several "standard" images for you.
-If you haven't already installed it, use `pkg> add TestImages`.
+If this is your first time working with images in julia, it's likely that you'll need to install some
```
Comment (johnnychen94) [inline docs/src/install.md]: FileIO no longer prompts to give installation suggestions, so I add this part at the beginning of this section.

File: docs/src/install.md [hunk 2]
```diff
@@ -62,17 +77,13 @@ display automatically:
 
 ![IJulia](assets/ijulia.png)
 
-Users of the Julia command-line interface (REPL) can install the [ImageView](https://github.com/timholy/ImageView.jl) package:
```
Comment (johnnychen94) [inline docs/src/install.md]: Not sure if removing this part is wanted.

File: docs/src/quickstart.md [hunk 3]
```diff
@@ -301,16 +301,6 @@ it straightforward to keep track of the correspondence between
 location across multiple images. More information can be found in
 [Keeping track of location with unconventional indices](@ref).
 
-## Display
```
Comment (johnnychen94) [inline docs/src/quickstart.md]: Moved to install.md

File: docs/src/install.md [hunk 4]
```diff
@@ -1,57 +1,72 @@
...
-using FileIO
-img = load("myphoto.png")
+(v1.0) pkg> add TestImages
 ```
 
-This should load the image for you, possibly prompting you to install
-an input/output package appropriate for your platform.
+To load one of the images from this package, say
+
+```julia
```
Comment (johnnychen94) [inline docs/src/install.md]: use `julia` to not display the same image twice.

File: docs/src/install.md [hunk 5]
```diff
@@ -1,57 +1,73 @@
...
-that these commands will prompt you to install one or more additional
-packages appropriate for your platform; you should generally accept
-the recommendation, unless you have reasons to prefer an alternate
-solution.
+and to load an image, you can use
+
+```@example
+using FileIO
+using ImageShow # hide
+img = load(joinpath("assets", "installation", "mandrill.tiff"))
```
Comment (timholy) [inline docs/src/install.md]: From which directory should this be run? It's useful as an internal test but I am less sure it's useful to users.

File: docs/src/install.md [hunk 6]
```diff
@@ -62,17 +78,13 @@ display automatically:
...
-using TestImages, Images, ImageView
-img = testimage("mandrill")
-imshow(img)
-```
+Currently there're five julia packages can be used to display an image:
 
-`ImageView` includes interactive features (panning/zooming, contrast
-adjustment, playing movies, labeling, etc.) and may be of interest
-even for users of graphical environments.
+* [`ImageShow`](https://github.com/JuliaImages/ImageShow.jl) is used to support image display in Juno and IJulia. This is automatically used when you use `Images`.
```
Comment (timholy) [inline docs/src/install.md]:
```suggestion
* [`ImageShow`](https://github.com/JuliaImages/ImageShow.jl) is used to support image display in Juno and IJulia. This happens automatically if you are `using Images`.
```

**Comment by johnnychen94** [comment]
> Maybe add a CONTRIBUTING.md file to this repo and post the link for inspecting the previews?

IIUC previews aren't built for PRs from other forks

File: docs/src/install.md (hunk 5 above)
Comment (johnnychen94) [reply docs/src/install.md]: The new commit hides the real image path, instead, it shows an apparently invalid path to tell users to specify their own ones.

```julia
using FileIO
using ImageShow # hide
# specify the path to your local image file
img_path = "/path/to/image.png"
img_path = joinpath("assets", "installation", "mandrill.tiff") # hide
img = load(img_path)
```

**Pull request merged**